source/Calamari/ArgoCD/Conventions/UpdateArgoCDApplicationManifestsInstallConvention.cs

```diff
@@ -146,26 +139,37 @@ public void Install(RunningDeployment deployment)
                                           .GetResult();
 
                                 didUpdateSomething = true;
+                                updatedApplications.Add(application.Name);
+                                gitReposUpdated.Add(applicationSource.RepoUrl.AbsoluteUri);
                             }
                             else
                             {
                                 log.Info("No changes were commited");
                             }
                         }
                     }
-                    
-                    //if we have links, use that to generate a link, otherwise just put the name there
-                    var linkifiedAppName = instanceLinks != null
-                        ? log.FormatLink(instanceLinks.ApplicationDetails(application.Name, application.KubernetesNamespace), application.Name)
-                        : application.Name;
+                }
+                
+                //if we have links, use that to generate a link, otherwise just put the name there
+                var linkifiedAppName = instanceLinks != null
+                    ? log.FormatLink(instanceLinks.ApplicationDetails(application.Name, application.KubernetesNamespace), application.Name)
+                    : application.Name;
 
-                    var message = didUpdateSomething
-                        ? "Updated Application {0}"
-                        : "Nothing to update for Application {0}";
+                var message = didUpdateSomething
+                    ? "Updated Application {0}"
+                    : "Nothing to update for Application {0}";
 
-                    log.InfoFormat(message, linkifiedAppName);
-                }
+                log.InfoFormat(message, linkifiedAppName);
```

Comment:
This accidentally got moved into the application source loop in https://github.com/OctopusDeploy/Calamari/pull/1607/files#diff-f2001e293f9cc44a50b1ae75abdf33f309517f3b59fbf7de0767a918208c55e2R145